autogluon/scheduler/hyperband.py

```diff
@@ -219,6 +219,10 @@ def _run_reporter(self, task, task_job, reporter, searcher, terminator):
         last_updated = None
         while not task_job.done():
             reported_result = reporter.fetch()
+            if 'traceback' in reported_result:
+                logger.exception(reported_result['traceback'])
```

Comment:
Please insert
   terminator.on_task_remove(task)
here. This is necessary for the terminator to keep its datastructures in order.

Reply:
done